### PR TITLE
Add list-lsps snippets

### PR DESCRIPTION
**Pull request by ok300**
(Re-)add snippets for the `list_lsps` service method.

### CODE REVIEW

**Review by JssDWt — APPROVED** [review]
LGTM

**Review by dangeross — APPROVED** [review]

File: snippets/dart_snippets/lib/connecting_lsp.dart [hunk 1]
```diff
@@ -10,6 +10,12 @@ Future<void> getLspInfo() async {
   // ANCHOR_END: get-lsp-info
 }
 
+Future<void> listLsps() async {
+  // ANCHOR: list-lsps
+  List<LspInformation> availableLsps = await BreezSDK().listLsps();
+  // ANCHOR_END: list-lsps
+}
```
Comment (erdemyerebasmaz) [inline snippets/dart_snippets/lib/connecting_lsp.dart:17]: return availableLsps after ANCHOR_END change fn type to List<LspInformation> to pass CI

Comment (erdemyerebasmaz) [reply snippets/dart_snippets/lib/connecting_lsp.dart:17]: I'm guessing same applies to Swift example.

File: snippets/go/connecting_lsp.go [hunk 2]
```diff
@@ -16,6 +16,14 @@ func GetLspInfo() {
 	// ANCHOR_END: get-lsp-info
 }
 
+func ListLsps() {
+	// ANCHOR: list-lsps
+	if err := sdk.ListLsps(); err != nil {
+		log.Printf("%#v", err)
+	}
```
Comment (erdemyerebasmaz) [inline snippets/go/connecting_lsp.go:23]:
```suggestion
	if lspList = err := sdk.ListLsps(); err != nil {
		log.Printf("%#v", err)
	}
```

Comment (dangeross) [reply snippets/go/connecting_lsp.go:23]: Sorry I missed this, I think this is more functional

```suggestion
	if lspList, err := sdk.ListLsps(); err == nil {
		log.Printf("%#v", lspList)
	}
```
or if the func has to be functional
```
func ListLsps() ([]breez_sdk.LspInformation, error) {
    // ANCHOR: list-lsps
    lspList, err := sdk.ListLsps()
    if err != nil {
	return []breez_sdk.LspInformation{}, err
    }
    // ANCHOR_END: list-lsps
    return lspList, nil
}
```

Comment (ok300) [reply snippets/go/connecting_lsp.go]: Done, thanks!

File: snippets/dart_snippets/lib/connecting_lsp.dart (hunk 1 above)
Comment (ok300) [reply snippets/dart_snippets/lib/connecting_lsp.dart:18]: Done

**Review by roeierez — APPROVED** [review]
LGTM

**Review by ubbabeck — APPROVED** [review]
Looks good to me!

**Pull request merged**